### ISSUE TITLE
In the CentOS system, the restore process needed to be executed twice can success.

### ISSUE THREAD

**Issue by lizhi16** [checkpoint-restore/criu#447]
My system is centos7 and the kernel is 3.10.0.x86_64. 
docker Version is 17.09.0-ce

My Question:
 when I used the "docker start --checkpoint-dir" to restore the checkpoint file, an error happened.
(01.427150) mnt: Switching to new ns to clean ghosts
(01.427224) Unlink remap tmp/jar_cache5045841997833305098.tmp.cr.1.ghost
(01.427916) Unlink remap tmp/jar_cache352020530951016212.tmp.cr.2.ghost
(01.428970) Error (criu/cr-restore.c:2171): Restoring FAILED.

BUT, I execute "docker start --checkpoint-dir" for the second time, the error disappears, and the restore succeeds.

anybody could tell me why this happened?

**Comment by avagin** [comment]
Could you send a full restore log? There should be other errors.

**Comment by lizhi16** [comment]
[restore.txt](https://github.com/checkpoint-restore/criu/files/1765525/restore.txt)

**Comment by avagin** [comment]
```
(00.708360) cg: Restoring cgroup property value [lo 0
eno1 0
enp4s0f1 0
virbr0 0
virbr0-nic 0
docker_gwbridge 0
docker0 0
brqefca0742-01 0
vxlan-17 0
veth1c3ef12 0
veth1fe6837 0] to [net_prio,net_cls/docker/8b8fe13cf869b41e04bca39f06e048f0a93928d6d337907ddb4f3dd65f603843/net_prio.ifpriomap]
(00.708398) Error (criu/cgroup.c:1225): cg: Failed writing lo 0
```

**Comment by avagin** [comment]
Cc: @0x7f454c46

**Comment by 0x7f454c46** [comment]
Hi guys,

thanks for Ccing,
I believe @lizhi16 has an old version of CRIU. The fix for cgroup net_prio write was merged somewhere in v3.7 release. So, you can either update criu or build it from sources.
As a workaround, you can try to umount it.. https://www.kernel.org/doc/Documentation/cgroup-v1/net_prio.txt

**Comment by lizhi16** [comment]
@0x7f454c46 @avagin Thank you so much! I updated to v3.7 release, this problem is solved.
